src/c-writer.cc

```diff
@@ -177,6 +193,7 @@ class CWriter {
   void PopLabel();
 
   static char MangleType(Type);
+  static constexpr char MangleField(ModuleFieldType);
```

Comment:
Maybe add `constexpr` to `MangleType` for consistency here?

Reply:
Sure, done.